apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverviewForm.tsx

```diff
@@ -445,41 +452,68 @@ const SignedVerdictOverviewForm: React.FC<Props> = (props) => {
         </>
       )}
       {!workingCase.isMasked && (
-        <Box marginBottom={user?.role === UserRole.PROSECUTOR ? 7 : 15}>
-          {user?.role !== UserRole.STAFF && (
-            <>
-              <Box marginBottom={3}>
-                <PdfButton
+        <Box marginBottom={10}>
+          <Text as="h3" variant="h3" marginBottom={5}>
+            {formatMessage(m.caseDocuments)}
+          </Text>
+          <Box marginBottom={2}>
+            <Stack space={2} dividers>
+              {user?.role !== UserRole.STAFF && (
+                <PdfRow
                   caseId={workingCase.id}
                   title={formatMessage(core.pdfButtonRequest)}
                   pdfType="request"
                 />
-              </Box>
-              <Box marginBottom={3}>
-                <PdfButton
+              )}
+              {workingCase.type === CaseType.CUSTODY &&
+                workingCase.state === CaseState.ACCEPTED &&
+                isAcceptingCaseDecision(workingCase.decision) && (
```

Comment:
Do we need to check the state and the decision? I would think checking only the state would be sufficient.

Reply:
We need this, because we should not show the custody notice if the decision is alternative travel ban.